### PR TITLE
Add the GetIdentityContent Command

### PR DESCRIPTION
**Pull request by mcstoer**
Adds the GetIdentityContent command using the new GetIdentityContent Request.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,6 @@
   ],
   "dependencies": {
     "axios": "1.7.4",
-    "verus-typescript-primitives": "https://github.com/VerusCoin/verus-typescript-primitives.git"
+    "verus-typescript-primitives": "https://github.com/mcstoer/verus-typescript-primitives.git#getidentitycontent"
```
Comment (michaeltout) [inline package.json:48]: Changes have been merged into typescript primitives now so all that needs to be done is an update of the existing lib, should only affect the yarn.lock file.

**Review by michaeltout — CHANGES_REQUESTED** [review]
Looks good, thanks for the PR! Minor note for tests and then to also point the ref of verus-typscript-primitives back to the main repo, changes were merged there.

File: src/__tests__/live/rpc.test.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { VerusdRpcInterface } from '../../index'
 jest.setTimeout(10000)
 
 describe('Makes live API Verusd RPC calls', () => {
-  const verusd = new VerusdRpcInterface("iJhCezBExJHvtyH3fGhNnt2NhU4Ztkf2yq", "http://127.0.0.1:8000")
+  const verusd = new VerusdRpcInterface("VRSCTEST", "http://foo:bar@127.0.0.1:18843")
```
[the remark this answers is not in the document]
Comment (mcstoer) [reply src/__tests__/live/rpc.test.ts]: This has been undone. Thank you for letting me know.

File: package.json (hunk 1 above)
Comment (mcstoer) [reply package.json]: Updated. Thank you!

**Review by michaeltout — APPROVED** [review]

**Pull request merged**